docs/en/latest/discovery/nacos.md

```diff
@@ -124,7 +124,7 @@ $ curl http://127.0.0.1:9180/apisix/admin/stream_routes/1 -H 'X-API-KEY: edd1c9f
 
 #### Specify the namespace
 
-Example of routing a request with an URI of "/nacosWithNamespaceId/*" to a service which name, namespaceId "http://192.168.33.1:8848/nacos/v1/ns/instance/list?serviceName=APISIX-NACOS&namespaceId=test_ns" and use nacos discovery client in the registry:
+Example of routing a request with an URI of "/nacosWithNamespaceId/*" to a service whose name, namespaceId "http://192.168.33.1:8848/nacos/v1/ns/instance/list?serviceName=APISIX-NACOS&namespaceId=test_ns" and use nacos discovery client in the registry:
```

Comment:
"with" might be better here and for other places with similar changes.

Reply:
@Neilblaze Could you make this change throughout? I still see some uses of "whose".

Reply:
ah I see, okay I'll do it.